package.json

```diff
@@ -36,8 +36,8 @@
     "@babel/preset-env": "^7.12.1",
     "@babel/preset-typescript": "^7.12.1",
     "@percy/cypress": "^2.3.2",
-    "@typescript-eslint/eslint-plugin": "4.0.1",
-    "@typescript-eslint/parser": "4.0.1",
+    "@typescript-eslint/eslint-plugin": "^5.58.0",
```

Comment:
Do we need to update these packages?

Reply:
This is how I fixed some of the unused vars issues. Decorators were causing errors.